README.md

```diff
@@ -31,27 +29,37 @@ You can choose your own path among the following levels:
 
 **TASKS**
 + *All of the base requirements*
-+ Create and implement (using HTML, CSS and JS) a UI for listing the [short list of systems](https://github.com/SUSE/frontend-challenge/blob/master/systems-short-list.json) systems with a basic design
++ Use a JavaScript framework ([ReactJS](https://reactjs.org/) is highly recommended)
```

Comment:
I would say the use of React is mandatory

Reply:
Agreed, it makes sense for the purpose of the challenge :+1:

Reply:
Made ReactJS mandatory.